pull/context.go

```diff
@@ -200,3 +204,15 @@ type Reviewer struct {
 	Name    string
 	Removed bool
 }
+
+type Signature struct {
+	Type              SignatureType
+	Email             string
+	IsValid           bool
+	KeyID             string
+	Payload           string
+	Signature         string
+	Signer            string
+	State             string
+	WasSignedByGitHub bool
+}
```

Comment:
Historically, I've tried to only expose fields that were actually used and then add additional fields as needed. It looks like `WasSignedByGitHub`, `Email`, `Payload`, `Signature`, and maybe others are not used. What do you think about removing those for now?